server/svix-server/Cargo.toml

```diff
@@ -44,6 +44,7 @@ bb8-redis = "0.10.1"
 redis = { version = "0.21.5", features = ["tokio-comp"] }
```

Comment:
Maybe also upgrade axum to latest before this PR (if it's not already), just so we can make sure we don't have any issues.